lightning_uq_box/models/cards.py

```diff
@@ -47,7 +47,7 @@ def forward(self, x: Tensor, t: Tensor) -> Tensor:
 class DiffusionSequential(nn.Sequential):
     """My Sequential to accept multiple inputs."""
 
-    def forward(self, input: Tensor, t: Tensor):
+    def forward(self, input: Tensor, t: Tensor) -> Tensor:  # type: ignore[override]
```

Comment:
`nn.Sequential` only accepts one input. We accept 2, breaking some expectations for subclasses, but otherwise harmless to ignore.